### ISSUE TITLE
Extra Children Appearing in Production Sidebar Despite any Configuration

### ISSUE THREAD

**Issue by Jstevenson02** [shuding/nextra#2839]
###  I resolved a very frustrating bug which occurred on Friday 

I raise this issue so that if anyone else happens across this issue they may find my solution.

I had created a documentation website using shuding's boilerplate docs repo on the Nextra offical documentation. Through following the docs I create a small tree of folders creating sidebar structure.

Although to my surprise, seemingly, running the production build locally and running the development build locally produced no errors.

Though when hosted on Vercel I saw extra children under my navigation elements. When inspected, they were styled A tags with no href, nothing more.

Couldn't for the life of me figure out why they appeared on production on Vercel but not production locally.

TLDR:  Delete the **.next** folder locally, commit to git, then create a **new** Vercel project deployment. Creating a new project deployment ensures no build is cached. This solve my issue.
